test/app-luatest/console_debugger_session_test.lua

```diff
@@ -290,36 +279,21 @@ local function run_debug_session(cmdline, sequence, tmpfile, header)
                 if cmd ~= '' then
                     local key, arg = get_key_arg_pair(cmd)
                     arg = arg and (' ' .. arg) or ''
-                    cmd = get_cmd_alias(key, i) .. arg .. '\n'
-                else -- '' empty command - repeat prior one
-                    cmd = cmd .. '\n'
+                    cmd = get_cmd_alias(key, i) .. arg
                 end
-                fh:write(cmd)
-                debuglog('Execute command: "'..trim(cmd)..'"')
+                debuglog('Execute command: "' .. trim(cmd) .. '"')
+                child:execute_command(cmd)
             end
 
-            local result
-            local clean_cmd = trim(cmd)
-            -- there should be empty stderr - check it before stdout
-            local errout = fh:read({ timeout = 0.05, stderr = true})
-            debuglog('stderr output: ', trim(errout))
-            if first and errout then
-                -- we do not expect anything on stderr
-                -- with exception of initial debugger header
-                t.assert_str_contains(trim(errout), header, false)
-                first = false
-            else
-                t.assert(errout == nil or trim(errout) == '')
-            end
```

Comment:
This commit is marked as refactoring, but actually it eliminates the stderr check. I don't know, whether it is important, but it should be mentioned in the commit message at least. If it is not important, the commit message should describe a reason.

Reply:
Added a remark in commit msg.

Reply:
OK. I'm relying on your investigation here.